sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -400,7 +399,7 @@ export class StudentService extends RecordDataModelService<Student> {
       studentToken.givenNames !== studentToSync.user.firstName
     ) {
       const sinValidation = new SINValidation();
-      studentToSync.birthDate = getDateOnly(studentToken.birthdate);
+      studentToSync.birthDate = new Date(studentToken.birthdate);
```

Comment:
do we need to remove `getDateOnly`

Reply:
Yes as mentioned in the previous comment.